src/algebra/gcd_monoid/basic.lean

```diff
@@ -518,6 +518,22 @@ theorem exists_eq_pow_of_mul_eq_pow [gcd_monoid α] [unique (units α)] {a b c :
   (h : a * b = c ^ k) : ∃ (d : α), a = d ^ k :=
 let ⟨d, hd⟩ := exists_associated_pow_of_mul_eq_pow hab h in ⟨d, (associated_iff_eq.mp hd).symm⟩
 
+lemma gcd_greatest {α : Type*} [comm_cancel_monoid_with_zero α] [normalized_gcd_monoid α]
+  {a b d : α} (hda : d ∣ a) (hdb : d ∣ b)
+  (he : ∀ e : α, e ∣ a → e ∣ b → e ∣ d) : gcd_monoid.gcd a b = normalize d :=
```

Comment:
Feel free to ignore but to me `he` reads as "hypothesis about `e`"
```suggestion
  (hd : ∀ e : α, e ∣ a → e ∣ b → e ∣ d) : gcd_monoid.gcd a b = normalize d :=
```

Reply:
Good point, I've changed it in all three lemmas.  Thanks!